whitepaper.md

```diff
@@ -0,0 +1,170 @@
+
+Sawtooth provides three consensus implementations, with a forth under active development:
+
+* **Dev_mode**: a simplified random leader algorithm that is useful for developers and test networks that require only crash fault tolerance.
+
+* **PoET**: short for "Proof of Elapsed Time", PoET is a Nakamoto-style consensus algorithm. It is designed to be a production-grade protocol capable of supporting large network populations that include byzantine actors. More information can be found about the PoET algorithm here - [PoET 1.0 Specification](https://sawtooth.hyperledger.org/docs/core/releases/latest/architecture/poet.html).
+
+* **PoET-Simulator**: Sawtooth includes an implementation of PoET that simulates the secure instructions. This mechanism enables large-scale consensus populations, but forgoes Byzantine fault tolerance.
+
+* **RAFT** (Under Development): The RAFT consensus protocol is a crash-fault tolerant ‘voting’-style consensus algorithm. It is designed for high throughput, low latency transactions. 
+
+**Privacy and Confidentiality**
+
+Blockchain privacy and confidentiality is one of the key challenges to blockchain adoption in the enterprise. To-date there have been two general means of providing privacy on a blockchain:
+
+1. **Separation of Concerns **- Data is only sent to the relevant parties of a transaction, with an optional hash of that data broadcast to all blockchain nodes. This means of privacy has been implemented by R3’s Corda, JPMorgan’s Quorom, and Hyperledger Fabric through it’s channels model. While this method achieves the privacy and data locality concerns, it has a few drawbacks. The key disadvantage of this model is that it creates a number of ‘sub-chains’ and transferring assets between these chains adds significant complexity or reduces confidentiality.
+
+2. **Computation on Encrypted Data** - In this approach, encrypted data is broadcast across the entire network in a way that the data can be computed on. This may entail advanced cryptographic techniques such as homomorphic encryption or zero-knowledge proofs, or leverage a trusted execution environment such as Intel® Software Guard Extensions (Intel® SGX). This method too comes with several drawbacks. Such drawbacks may include the need for trusted hardware or performance and usability limitations with advanced cryptographic techniques.
```

Comment:
Even though it is okay to end a sentence with a preposition, I suggest changing "in a way that the data can be computed on" to something like "in a way that allows computation on the data".